### PR TITLE
Enhancement: 0.8.0 Release Announcement

### PR DESCRIPTION
**Pull request by BenjamenMeyer**
Release announcement

### CODE REVIEW

**Review by Loki1950 — APPROVED** [review]
Looks good @BenjamenMeyer

**Review by Loki1950 — APPROVED** [review]

**Review by Loki1950 — APPROVED** [review]

**Review by Loki1950 — APPROVED** [review]

File: _posts/2022-03-24-vegastrike-0.8.0-release.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+"LEGACY" builds; while the newer support is provided using the "GLVND" builds.
+
+The Vega Strike Engine now supports a Versioning API that provides the release version (e.g 0.8.0) and the Engine API Version (e.g 1).
+The Engine API version works much like the Android API Version - it's a simple incrementing unsigned integer value. If the API does
+not exist (import error in Python) then the Engine API Version is zero (0) and the release version is at most 0.7.0, but can also
+be any earlier version. We highly recommend that game assets developers gate on this API as going forward we are going to start
+introducing changes where games will need to rely on the Engine API Version value to determine that they are compatible with the
+engine. Aside from the introduction of this API there is no difference in the Python API provided from 0.7.0.
+
+The Vega Strike Developers are also putting in place a standard binary naming convension of `vegastrike-<tool` or `libvegastrike-<library>`.
```
Comment (stephengtuggy) [inline _posts/2022-03-24-vegastrike-0.8.0-release.md:35]: `convention` not `convension`. Also, the closing angle bracket is missing after `tool`.

File: assets/installation.md [hunk 2]
```diff
@@ -13,11 +13,15 @@ Below is a listing of installation for each platform.
 We have not yet restored building the latest versions for Windows.
 Please see the [Vega Strike 0.5.1](https://sourceforge.net/projects/vegastrike/files/vegastrike/0.5.1/) for a release to use.
 
+NOTE: Version 0.8.0 has reestored building on Windows. Pre-built binaries are not yet being provided at this time.
```
Comment (stephengtuggy) [inline assets/installation.md:16]: `restored` not `reestored`

File: assets/installation.md [hunk 3]
```diff
@@ -13,11 +13,15 @@ Below is a listing of installation for each platform.
...
+
 ## Mac OS X
 
 We have not yet restored building the latest versions for Mac OS X.
 Please see the [Vega Strike 0.5.0](https://sourceforge.net/projects/vegastrike/files/vegastrike/0.5.0/) for a release to use.
 
+NOTE: Version 0.8.0 has reestored building on Mac OS X. Pre-built binaries are not yet being provided at this time.
```
Comment (stephengtuggy) [inline assets/installation.md:23]: `restored` not `reestored`, as above

**Review by stephengtuggy — CHANGES_REQUESTED** [review]
Just a couple of typos I noticed. Everything else looks great.

File: assets/installation.md (hunk 3 above)
Comment (Loki1950) [reply assets/installation.md:23]: Must remember that Github does not do auto-correct ;-)

**Review by Loki1950 — APPROVED** [review]

**Review by stephengtuggy — APPROVED** [review]
Perfect.

**Pull request merged**